qiskit/extensions/quantum_initializer/initializer.py

```diff
@@ -42,7 +42,7 @@ class Initialize(Instruction):
     which is not unitary.
     """
 
-    def __init__(self, params):
+    def __init__(self, params, num_qubits=None):
```

Comment:
Can you add a comment and error message that this argument is only valid if `params` is an integer? To capture cases where users provide e.g. labels and the number of qubits.

Reply:
Right. I forgot to add the int and num_qubits options to the doc string. Also added an error if not integer `params `and `num_qubits ` not None.

Done in https://github.com/Qiskit/qiskit-terra/pull/5418/commits/8a7f33f292af59659e661aff949c9e31a25ec79a